tensorflow/compiler/mlir/tools/kernel_gen/transforms/passes.td

```diff
@@ -51,8 +51,12 @@ def TFToJITInvocationPass : FunctionPass<"tf-to-jit-invocation"> {
              /*default=*/"", "Max rank that this kernel supports">,
       Option<"enable_ftz_", "enable-ftz", "bool", /*default=*/"",
              "Enable the denormal flush to zero mode when generating code">,
+      Option<"index_64bit_", "index_64bit", "bool", /*default=*/"",
+             "Enable the 64 bit indexing for GPU kernels">,
       Option<"cpu_codegen_", "cpu-codegen", "bool", /*default=*/"",
              "CPU codegen (false implies GPU)">,
+      Option<"jit_i64_indexed_for_large_tensors_", "jit_i64_indexed_for_large_tensors", 
+      "bool", /*default=*/"true", "Enable the AOT JIT kernel gen path">,
```

Comment:
Can we concretize the description? E.g.: 
Enable JIT compilation of i64-indexed kernels for large input tensors.

Reply:
fixed.